webpack/assets/javascripts/react_app/components/Layout/components/UserDropdowns.scss

```diff
@@ -0,0 +1,81 @@
+@import '~patternfly/dist/sass/patternfly/_color-variables.scss';
+
+.masthead-icon {
+  margin-top: -15px;
+  padding-top: 5px;
+}
+
+@-moz-keyframes blink {
+  0% {
+    opacity: 1;
+  }
+
+  50% {
+    opacity: 0;
+  }
+
+  100% {
+    opacity: 1;
+  }
+}
+
+@-webkit-keyframes blink {
+  0% {
+    opacity: 1;
+  }
+
+  50% {
+    opacity: 0;
+  }
+
+  100% {
+    opacity: 1;
+  }
+}
+
+/* IE */
+@-ms-keyframes blink {
+  0% {
+    opacity: 1;
+  }
+
+  50% {
+    opacity: 0;
+  }
+
+  100% {
+    opacity: 1;
+  }
+}
+
+/* Opera and prob css3 final iteration */
+@keyframes blink {
```

Comment:
I believe that today all of the browsers can read the `@keyframes` so you don't really need to add specific browser prefixes.